### PR TITLE
Rename service to application

### PR DESCRIPTION
**Pull request by wallyworld**
Services are now Applications

### CODE REVIEW

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/jujusvg/72/
Test PASSed.

File: canvas.go [hunk 1]
```diff
@@ -13,42 +13,42 @@ import (
...
-	services      []*service
-	relations     []*serviceRelation
+	applications  []*application
+	relations     []*applicationRelation
 	iconsRendered map[string]bool
 	iconIds       map[string]string
 }
 
-// service represents a service deployed to an environment and contains the
+// application represents a application deployed to an environment and contains the
```
Comment (axw) [inline canvas.go]: to a model

**Comment by axw** [comment]
Looks fine, but please get the OK from @makyo or someone else on the UI team. There may be some CSS changes required.

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/jujusvg/73/
Test PASSed.

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/jujusvg/74/
Test PASSed.

**Comment by mhilton** [comment]
:+1:

**Comment by makyo** [comment]
👍

**Comment by rogpeppe** [comment]
LGTM except that this is a backwardly incompatible change in stably-versioned repository. We should move to v2.

**Comment by mhilton** [comment]
Replaced by #51

**Pull request closed without merging**